<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="Shashidhar" failures="1" tests="1" name="Test" time="1.715" errors="0" timestamp="2023-07-13T23:35:41 IST">
  <testcase classname="github_test_cases.github_testcases" name="CreateRepoDynamic" time="1.686">
    <failure type="java.lang.AssertionError" message="expected [null] but found [Shashidhar_Restassured_framework_repo]">
      <![CDATA[java.lang.AssertionError: expected [null] but found [Shashidhar_Restassured_framework_repo]
at github_test_cases.github_testcases.CreateRepoDynamic(github_testcases.java:51)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- CreateRepoDynamic -->
  <testcase classname="github_test_cases.github_testcases" name="createRepository" time="0.0">
    <ignored/>
  </testcase> <!-- createRepository -->
</testsuite> <!-- Test -->
